### ISSUE TITLE
Add green dot for logged in users and grey for others instead repetitive text

### ISSUE THREAD

**Issue by anandkumar** [elementary/wingpanel-indicator-session#49]
It is not a big deal but when I see all those "logged in" and "logged out" text below usernames bugs me. I propose to add a green dot for logged in user and grey dot for logged out users. 

The dot could be partially overlapping the profile photo or at the end of a user's name (aligned right).

**Comment by danrabbit** [comment]
We could probably add badges in addition to the text, but we want to make sure to keep color blind users in mind for whom color alone might not be helpful

**Comment by anandkumar** [comment]
Good point. Then I would suggest a green solid dot vs grey ring. It would solve the color blindness issue. My point is removing those text.

**Comment by jwinnie** [comment]
@anandkumar @danrabbit Or you could just have no badge for users who aren’t logged in.

**Comment by cassidyjames** [comment]
On the greeter we just use a :heavy_check_mark: in a circle for logged-in users. We could overlay that here as well, but honestly removing the status text will make the design a little unbalanced.
